### PR TITLE
module_utils/redfish_utils.py: add ForceRestart option

### PR DESCRIPTION
**Pull request by digifuchsi**
##### SUMMARY

The library does not allow you to force a restart via redfish (unless GracefulRestart triggered by PowerReboot-command is not supported).
Therefore, I added the option PowerForceRestart to be able to force a reboot of an unresponsive host.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

redfish_utils.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/vmware/vmware_cluster_vsan_stretched.py [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 VMware, Inc.
+# SPDX-License-Identifier: GPL-3.0-or-later
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.vmware import PyVmomi, vmware_argument_spec, find_hostsystem_by_name, find_cluster_by_name
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/vmware/vmware_cluster_vsan_stretched.py]: Could you please create a separate PR for this module ?

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/vmware/vmware_cluster_vsan_stretched.py (hunk 1 above)
Comment (digifuchsi) [reply lib/ansible/modules/cloud/vmware/vmware_cluster_vsan_stretched.py]: That was a PEBCAK...
This module should be in a different branch only...

**Comment by mraineri** [comment]
shipit

**Comment by billdodd** [comment]
shipit

**Pull request merged**